wiz-kubernetes-connector/templates/job-create-connector.yaml

```
@@ -0,0 +1,87 @@
+  backoffLimit: 5
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "wiz-kubernetes-connector.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ .Values.autoCreateConnector.serviceAccount.name }}
+      restartPolicy: "Never"
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: wiz-connector-creator
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: IfNotPresent
+          command: ["wiz-broker"]
+          args: [
+            "create-kubernetes-connector",
+            "--api-server-endpoint", 
+            {{ include "wiz-kubernetes-connector.apiServerEndpoint" . | trim | quote }},
+            "--secrets-namespace", 
+            {{ .Release.Namespace | quote }},
+            "--service-account-token-secret-name",
+            {{ include "wiz-kubernetes-connector.clusterReaderToken" . | quote }},
+            {{- if .Values.broker.enabled }}
+            "--output-secret-name",
+            {{ include "wiz-kubernetes-connector.connectorSecretName" . | trim | quote | nindent 12 }},
+            {{- end }}
+            "--is-on-prem={{ .Values.broker.enabled }}",
+            {{ with .Values.autoCreateConnector.connectorName }}
+            "--connector-name",
+            {{ . | quote }},
+            {{ end }}
+            ]
+          env:
+          - name: CREATE_KUBERNETES_CONNECTOR_CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "wiz-kubernetes-connector.apiTokenSecretName" . | trim }}
+                key: clientId
+          - name: CREATE_KUBERNETES_CONNECTOR_CLIENT_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "wiz-kubernetes-connector.apiTokenSecretName" . | trim }}
+                key: clientToken
+          - name: CREATE_KUBERNETES_CONNECTOR_ENV
+            value: {{ .Values.wizApiToken.clientEndpoint | quote }}
+          {{- if .Values.httpProxyConfiguration.enabled }}
+          - name: HTTP_PROXY
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "wiz-kubernetes-connector.proxySecretName" . | trim }}
+                key: http-proxy
+          - name: HTTPS_PROXY
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "wiz-kubernetes-connector.proxySecretName" . | trim }}
+                key: https-proxy
+          - name: NO_PROXY
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "wiz-kubernetes-connector.proxySecretName" . | trim }}
+                key: no-proxy-address
```

Let's align to use camelCase like in the clientId / clientToken ?

```suggestion
                key: httpProxy
          - name: HTTPS_PROXY
            valueFrom:
              secretKeyRef:
                name: {{ include "wiz-kubernetes-connector.proxySecretName" . | trim }}
                key: httpsProxy
          - name: NO_PROXY
            valueFrom:
              secretKeyRef:
                name: {{ include "wiz-kubernetes-connector.proxySecretName" . | trim }}
                key: noProxyAddress
```